src/helpers/articles/rateArticleHelper.js

```diff
@@ -44,6 +44,24 @@ class ArticleRatelehelper {
     }
     return true;
   }
+
+  /**
+     * Check the environment
+     * @function getRatings
+     * @param  {integer} article - Check the articleId
+     * @return {string} Validate the rate
+     */
+  static async getRatings(article) {
+    const ratings = await Rating.findAll({ where: { articleId: article } });
+
+    let sum = 0;
+    ratings.map((rate) => {
```

Comment:
`Arrays.prototype.map` is not the appropriate method to use here. This is because map returns a new list. `Array.prototype.forEach` is more appropriate